### DISCUSSION TITLE
Can an atom hold a component?

### DISCUSSION THREAD

**Discussion by AjaxSolutions** [pmndrs/jotai#1342] [Q&A] [answered]
Can the useAtom hook return a component and when (if ever) would this pattern make sense?

See this YouTube video: [How to return components from React hooks](https://www.youtube.com/watch?v=-UjbTFc9NL4)

**Comment by dai-shi** [comment, marked as answer]
See this for example: https://twitter.com/dai_shi/status/1468875799327817730
